### DISCUSSION TITLE
apple m1 compatible build

### DISCUSSION THREAD

**Discussion by alfanse** [BurntSushi/ripgrep#2291] [General]
I have the luck of working for an organisation where "we're rolling out m1 macs now and the x86_64 macs have been deprecated" as such I'm hunting for apple m1 compatible binaries of ripgrep.

Is it on the cards?

**Comment by BurntSushi** [comment]
I'm sure it's in the cards, yes. There's even a PR for it: https://github.com/BurntSushi/ripgrep/pull/2099

The bottom line though is that I have to integrate it into the release and make sure it's working. That PR is, IMO, in a bit of a scattered state.

But this is just a release binary. It should compile fine or be available in Homebrew I imagine.

Reminder: I don't use macOS. (I have a mac mini for testing things occasionally, but that's it.)
